modules/django-articles/articles/models.py

```diff
@@ -2,7 +2,7 @@
 from django.db import models
 from modules.utils import get_options
 
-MEDIA_UPLOAD_PATH = get_options("django-articles", "MEDIA_UPLOAD_PATH")
+MEDIA_UPLOAD_PATH = get_options("django_articles", "MEDIA_UPLOAD_PATH")
```

Comment:
The other was OK, here it should be the module slug because is how options are generated in the backend, see https://github.com/crowdbotics/modules/blob/modules_as_pkgs/docs/authoring.md#the-python-package-content

Reply:
Okay let me rebase and drop it, thanks.